Git.md

```diff
@@ -37,3 +34,46 @@ doc/*.txt
 doc/**/*.pdf
 
 ```
+
+## <a name="creatingBranches">Create Branch</a>
+
+1. First, check in which branch you are right now:   
+```
+git branch
+```
+You'll get a list of the branchs available, the branch with an ```*``` next to it, is the branch you are working on.
+
+2. Then, create a new branch:
+```
+git branch name_of_the_branch_to_create
+```
+
+3. Move to the new branch created:
+```
+git checkout name_of_the_branch_to_create
+```
+
+4. You can do step 2 and 3 by one command:
+```
+git checkout -b name_of_the_branch_to_create
+
+-----------
+
+## <a name="modifyingCommits">[Modifying Commits](https://classroom.udacity.com/courses/ud123/lessons/f02167ad-3ba7-40e0-a157-e5320a5b0dc8/concepts/e176503b-3eae-4b22-a1b3-2953bab3d5e5)</a>
```

Comment:
Check if the preview for these lines of code below is ok

Reply:
I added '```' 😅

Corrected